### PR TITLE
catch stderr from yarn subprocess, addresses #2301

### PR DESCRIPTION
**Pull request by benharri**

### CODE REVIEW

File: src/command-line/install.js [hunk 1]
```diff
@@ -90,6 +90,10 @@ program
 				});
 			});
 
+			add.stderr.on("data", (data) => {
+				log.error(data);
```
Comment (xPaw) [inline src/command-line/install.js]: Does it work without `toString`?

Comment (benharri) [reply src/command-line/install.js]: Good call. I added `.toString()`

**Comment by xPaw** [comment]
Can you rebase your PR correctly?

**Comment by benharri** [comment]
Fixed the rebase. Should be good to go.

**Comment by benharri** [comment]
Squashed!

**Pull request merged**